mods/ra/lua/ai.lua

```diff
@@ -221,56 +220,57 @@ MOD_choose_build_order = function(faction)
 end
 
 MOD_is_soviet = function(faction_name)
-  if faction_name == "soviet" or faction_name == "ukraine" or faction_name == "soviet" then
-    return true
-  else
-    return false
-  end
+	if faction_name == "soviet" or faction_name == "ukraine" or faction_name == "soviet" then
```

Comment:
Why is `faction_name == "soviet"` listed twice? Is one of them supposed to be "russia"?